saleor/payment/gateways/razorpay/forms.py

```diff
@@ -0,0 +1,69 @@
+from django import forms
+from django.forms.utils import flatatt
+from django.forms.widgets import HiddenInput
+from django.utils.html import format_html
+from django.utils.translation import pgettext_lazy
+
+from ...forms import PaymentForm
+
+CHECKOUT_SCRIPT_URL = 'https://checkout.razorpay.com/v1/checkout.js'
+
+PRIMARY_BTN_TEXT = pgettext_lazy(
+    'Rozorpay payment gateway primary button', 'Pay now with Razorpay')
+SECONDARY_TITLE = pgettext_lazy(
+    'Rozorpay payment gateway secondary title', 'Total payment')
+
+
+class RazorPayCheckoutWidget(HiddenInput):
+    def __init__(
+            self,
+            *, payment,
+            public_key, prefill, store_name, store_image,
+            **kwargs):
+        override_attrs = kwargs.get('attrs', None)
+        base_attrs = kwargs['attrs'] = {
+            'src': CHECKOUT_SCRIPT_URL,
+            'data-key': public_key,
+            'data-buttontext': PRIMARY_BTN_TEXT,
+            'data-image': store_image,
+            'data-name': store_name,
+            'data-description': SECONDARY_TITLE,
+            'data-amount': int(payment.total * 100),
```

Comment:
I see this operation all over the files, can we pack it into some kind of a function, like `get_amount_for_razorpay` and add a nice docstring why is that?